src/components/MultiHopTrade/components/TradeInput/components/SellAssetInput.tsx

```diff
@@ -40,10 +45,7 @@ export const SellAssetInput = memo(
     // this is separated from handleSellAssetInputChange to prevent resetting the input value to 0
     // when the market data changes between keystrokes
     const handleSellAssetInputChangeInner = useCallback(
-      (sellAmountUserCurrencyHuman: string, sellAmountCryptoPrecision: string) => {
-        setSellAmountUserCurrencyHuman(sellAmountUserCurrencyHuman)
-        setSellAmountCryptoPrecision(sellAmountCryptoPrecision)
-
+      (sellAmountCryptoPrecision: string) => {
```

Comment:
Will need to test this ensure this doesn't break fiat input / fiat from crypto prorating

Reply:
Confirmed happy 🤌